src/core/include/ngraph/op/roi_align.hpp

```diff
@@ -12,6 +12,9 @@ namespace op {
 namespace v3 {
 using ov::op::v3::ROIAlign;
 }  // namespace v3
+namespace v9 {
+using ov::op::v9::ROIAlign;
+}  // namespace v9
 using v3::ROIAlign;
```

Comment:
why do we need L18? Default v3? Looks we can remove it.

Reply:
as we synced offline, this will keep previous namespace op api unchanged.